datashader/tiles.py

```diff
@@ -92,7 +94,11 @@ def render_super_tile(tile_info, output_path, load_data_func,
     agg = rasterize_func(df, x_range=tile_info['x_range'],
                          y_range=tile_info['y_range'], height=tile_size,
                          width=tile_size)
-    ds_img = shader_func(agg, span=tile_info['span'])
+    # if the rasterized data is empty, do not create subtiles.
+    if np.sum(agg) == 0:
+        pass
```

Comment:
Won't this just error because `ds_img` won't be defined below?

Reply:
```suggestion
        return
```
?

Reply:
@jbednar I would vote for closing this PR and not using the `np.sum(agg) == 0` heuristic because it feels brittle to me.

Reply:
Ok, will do.  Anyone who feels otherwise can reopen while addressing the above issue(s).